From 27690df52d51273fd67e148bf10077433b564692 Mon Sep 17 00:00:00 2001
From: Wei Yang <Wei.Yang@windriver.com>
Date: Tue, 23 Apr 2013 13:43:54 +0800
Subject: [PATCH] dts/p1023: Adjust the layout of nor flash

*) Adjust the layout of nor flash to suite the
   released uboot.

Signed-off-by: Wei Yang <Wei.Yang@windriver.com>
---
 arch/powerpc/boot/dts/p1023rdb.dts |   21 ++++++++++++---------
 1 files changed, 12 insertions(+), 9 deletions(-)

diff --git a/arch/powerpc/boot/dts/p1023rdb.dts b/arch/powerpc/boot/dts/p1023rdb.dts
index f32c836..f3f7505 100644
--- a/arch/powerpc/boot/dts/p1023rdb.dts
+++ b/arch/powerpc/boot/dts/p1023rdb.dts
@@ -136,22 +136,25 @@
 			bank-width = <2>;
 			device-width = <1>;
 			partition@0 {
-				label = "ramdisk";
-				reg = <0x00000000 0x01c40000>;
+				label = "fs1";
+				reg = <0x00000000 0x01000000>;
 			};
-			partition@1c40000 {
+			partition@1000000 {
+				label = "fman ucode";
+				reg = <0x01000000 0x00020000>;
+			};
+			partition@1020000 {
+				label = "fs2";
+				reg = <0x01020000 0x00c40000>;
+			};
+			partition@1c60000 {
 				label = "kernel";
 				reg = <0x01c40000 0x002e0000>;
 			};
-			partiton@1f20000 {
+			partiton@1f40000 {
 				label = "dtb";
 				reg = <0x01f20000 0x00020000>;
 			};
-			partition@1f40000 {
-				label = "firmware";
-				reg = <0x01f40000 0x00020000>;
-				read-only;
-			};
 			partition@1f60000 {
 				label = "u-boot env";
 				reg = <0x01f60000 0x00020000>;
-- 
1.7.5.4

